[README] clarify spec changes in ratification process (#603)
oneapi-src/oneMKL: README.md

4. Organize offline review or/and bring the RFC to [UXL Foundation Math SIG forum](https://lists.uxlfoundation.org/g/Math-SIG), [UXL Foundation Open Source Working Group](https://lists.uxlfoundation.org/g/open-source-wg), or any other related forums in order to collect feedback.
    * It's recommended to keep all feedback as part of PR review, so it also
will be documented in one place

> 5. If the changes affect the API defined by [oneMKL specification](https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onemkl/source/) a new [issue](https://github.com/uxlfoundation/oneAPI-spec/issues) with the \[RFC\] tag must be created to describe the API changes. The specification RFC must be reviewed by [UXL Foundation Math SIG forum](https://lists.uxlfoundation.org/g/Math-SIG), and only after it the proposed changes can be implemented in this project.

Reviewer: I didn't find `RFC` in the current [labels](https://github.com/uxlfoundation/oneAPI-spec/labels). Will it be created? Or does it mean the issue title should be prefixed by "[RFC]"?

nitpicking
```suggestion
5. If the changes affect the API defined by [oneMKL specification](https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onemkl/source/), a new [issue](https://github.com/uxlfoundation/oneAPI-spec/issues) with the \[RFC\] tag must be created to describe the API changes. The specification RFC must be reviewed by [UXL Foundation Math SIG forum](https://lists.uxlfoundation.org/g/Math-SIG), and only after it the proposed changes can be implemented in this project.
```

Author: It's actually not a label, but a tag in the issue name https://github.com/uxlfoundation/oneAPI-spec/issues?q=is%3Aissue+is%3Aopen+RFC but maybe we can ask for the introducing the new label in the spec.

> 5. If changes affect API defined by [oneMKL specification](https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onemkl/source/) the related part of the design document must be converted to oneAPI specification RFC (as a new [issue](https://github.com/uxlfoundation/oneAPI-spec/issues) with \[RFC\] tag), reviewed by [UXL Foundation Math SIG forum](https://lists.uxlfoundation.org/g/Math-SIG), and contributed to [oneAPI specification](https://github.com/uxlfoundation/oneAPI-spec), and only after it the proposed changes can be implemented in this project.

Reviewer 2: I find that long sentence confusing and I'm not sure "converted" is the right word in this context, "duplicated" would be more truthful. I would suggest:

```suggestion
5. If the changes affect the API defined by [oneMKL specification](https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onemkl/source/) a new [issue](https://github.com/uxlfoundation/oneAPI-spec/issues) with the \[RFC\] tag must be created to describe the API changes. The specification RFC must be reviewed by [UXL Foundation Math SIG forum](https://lists.uxlfoundation.org/g/Math-SIG), and only after it the proposed changes can be implemented in this project.
```

Author: Agree, the suggestion looks better, thank you!
